resources/function_help/json/array_minority

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "array_minority",
+  "type": "function",
+  "groups": ["Arrays"],
+  "description": "Return the less common values in an array.",
+  "arguments": [
+    {"arg":"array","description":"an array"},
+    {"arg":"option='all'", "optional":true, "description":"a string specifying the return values handling. Valid options are:<br /><ul><li>all: Default, all less common values are returned in an array</li><li>first: Return the first of the less common values</li><li>median: Return the median of the less common values</li><li>real_minority: Return all values except the real majority if thre is one</li></ul>"}
```

Comment:
```suggestion
    {"arg":"option='all'", "optional":true, "description":"a string specifying the return values handling. Valid options are:<br /><ul><li>all: Default, all less common values are returned in an array</li><li>first: Returns the first of the less common values</li><li>median: Returns the median of the less common values</li><li>real_minority: Returns values which occur less than half the size of the array</li></ul>"}
```
IMHO self-contained and avoids people needing to track what the real majority is.
And same question about what first refers to.